ds3-autogen-c/src/main/java/com/spectralogic/ds3autogen/c/helpers/RequestHelper.java

```diff
@@ -107,12 +107,28 @@ public static String generateInitRequestFunctionSignature(final Request request)
         builder.addAll(request.getOptionalQueryParams().stream()
                 .map(Parameter::toString)
                 .collect(GuavaCollectors.immutableList()));
+
+        if (request.hasRequestPayload()) {
```

Comment:
This can be simplified to not have nested ifs:

if (request.hasRequestPayload() 
&& (!request.getName().equalsIgnoreCase("put_object_request")  
&& !request.getName().equalsIgnoreCase("complete_multi_part_upload_request")) ) {
  builder.add(...)
}